File: library/src/com/devspark/sidenavigation/ISideNavigationCallback.java
package com.devspark.sidenavigation;

/**
 * Callback interface for {@link SideNavigationView}.
 * 
 * @author e.shishkin
 * 
 */
public interface ISideNavigationCallback {

    /**
     * Validation clicking on side navigation item.
     * 
     * @param itemId id of selected item
     */
    public void onSideNavigationItemClick(int itemId);

}

File: sample.net/SideNavigation/ISideNavigationCallback.cs
/**
 * 
 * @author hram
 *
 */
namespace com.devspark.sidenavigation
{
    public interface ISideNavigationCallback
    {

        /**
         * Validation clicking on side navigation item.
         * @param itemId id of selected item
         */
        void onSideNavigationItemClick(int itemId);
    }
}
